### PR TITLE
Added crate dolly for handling camera transform/rotation

### PR DESCRIPTION
**Pull request by louisgjohnson**
Introducing the crate([dolly](https://github.com/h3r2tic/dolly)) into hotham-simulator for handling the camera transform and rotation

### CODE REVIEW

File: hotham-simulator/src/state.rs [hunk 1]
```diff
@@ -216,7 +204,7 @@ impl State {
 
         let dt = delta.as_secs_f32();
         let movement_speed = 2f32 * dt;
-        let mouse_sensitivity = 4f32 * dt;
+        let mouse_sensitivity = 40f32 * dt;
```
Comment (louisgjohnson) [inline hotham-simulator/src/state.rs]: not sure why but I had to massive bump the mouse_sensitivity up

Comment (rasmusgo) [reply hotham-simulator/src/state.rs]: `dt` should not be here because we don't want our framerate to decide how fast the mouse turns the view.

Comment (louisgjohnson) [reply hotham-simulator/src/state.rs]: yeah good point, we probably don't really need it for camera directional movement either

**Comment by rasmusgo** [comment]
It seems like dolly is overkill for what we are asking it to do at the moment. We are not using any smoothing or putting the camera on an arm. Do you have any particular features in mind to motivate adding dolly (and indirectly glam)?

**Comment by kanerogers** [comment]
Ah hrm, that is a good point @rasmusgo , I didn't realise how opinionated `dolly` was (using another maths library is not good). 😦 

I'm really sorry I didn't look into `dolly` more before I suggested it! I don't want you to feel like you wasted your time on this @louisgjohnson - what do you think `dolly` gives us that we don't already have?

**Comment by kanerogers** [comment]
I'm going to close this one off for now! Sorry about that @louisgjohnson 😦

**Pull request closed without merging**